index.html

```diff
@@ -1613,20 +1608,72 @@ <h3>
-resolved.
+<code><a>id</a></code> property.
         </p>
 
       </section>
 
+      <section>
+       <h3>DID Controller</h3>
+
+        <p>
+A <a>DID controller</a> is an entity that is authorized to make changes to a
+<a>DID document</a>. The process of authorizing a <a>DID controller</a> is
+defined by the <a>DID Method</a>.
+        </p>
+
+        <dl>
+          <dt><dfn>controller</dfn></dt>
+          <dd>
+The <code>controller</code> property is OPTIONAL. If present, the value MUST
+be a <a data-cite="INFRA#string">string</a> or an <a
+data-cite="INFRA#ordered-set">ordered set</a> of <a
+data-cite="INFRA#string">strings</a> that conform to the rules in <a
+href="#did-syntax"></a>. The corresponding <a>DID document</a>(s) SHOULD
+contain <a>verification relationships</a> that explicitly permit the use of
+certain <a>verification methods</a> for specific purposes.
+          </dd>
+        </dl>
+
+        <p>
+When a <code><a>controller</a></code> property is present in a <a>DID
+Document</a>, its value expresses one or more <a>DIDs</a>. Any <a>verification
+methods</a> contained in the <a>DID Documents</a> for those <a>DIDs</a> SHOULD
+be accepted as authoritative, such that proofs that satisfy those
+<a>verification methods</a> are to be considered equivalent to proofs provided
+by the <a>DID Subject</a>.
+        </p>
+
+        <pre class="example nohighlight"
+          title="DID document with a controller property">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:example:123456789abcdefghi",
+  "controller": "did:example:bcehfew7h32f32h7af3",
+}
+        </pre>
+
+        <p class="note" title="Authorization vs authentication">
+Note that authorization provided by the value of <code>controller</code> is
+separate from authentication as described in <a href="#authentication"></a>.
+This is particularly important for key recovery in the case of cryptographic
+key loss, when the subject no longer has access to their keys, or key
+compromise, where the <a>DID controller</a>'s trusted third parties need to
+override malicious activity by an attacker. See
+<a href="#security-considerations"></a> for information related to threat
+models and attack vectors.
+        </p>
+      </section>
+
       <section>
         <h3>Also Known As</h3>
 
-        <p class="issue atrisk" data-number="584">
-This property is at risk of being removed from DID Core in favour of putting
-it in the DID Specification Registries [[?DID-SPEC-REGISTRIES]] pending
-feedback from implementers and reviewers.
+        <p class="issue atrisk" title="Implementation of alsoKnownAs">
+The DID Working Group is current seeking implementer feedback regarding
+the alsoKnownAs feature. If there is not enough
+implementer interest in implementing this feature, it will be removed
+from this specification and placed into the DID Specification Registries
+[[?DID-SPEC-REGISTRIES]] as an extension.
```

Comment:
Should this not use the same language as the other At Risk labels in the document, including specifying what is meant by "enough implementer interest", i.e., two independent and interoperable implementations of the feature?

Reply:
Yes, it should. I'll try to make another pass and tighten up all the at risk language to be more consistent in a separate PR.